test/cpp/api/optim.cpp

```diff
@@ -523,3 +523,18 @@ TEST(OptimTest, CheckLRChange_StepLR_Adam) {
 
   check_lr_change(optimizer, step_lr_scheduler, expected_epoch_lrs);
 }
+
+TEST(OptimTest, CheckLRChange_ReduceLROnPlateau_Adam) {
+  torch::Tensor parameters = torch::zeros({1});
+  auto optimizer = Adam({parameters}, AdamOptions().lr(1e-3));
+
+	const SchedulerMode scheduler_mode = SchedulerMode::min;
```

Comment:
nit: is the irregular spacing here intentional?

Reply:
No, can you finish the PR ? At first I just wanted to warn you that an include was missing, I do not want to do indentation, verify test pass etc.

Reply:
Ah, I see haha, thanks for the honesty. We'll have someone else pick up the work here--thanks for the warning!

Reply:
Thanks for understanding :)